cypress/support/step-definitions/themes-steps.js

```diff
@@ -57,3 +57,7 @@ Then('Loader component css background color is set to {string}', (themeName) =>
     loaderComponent().should('have.css', BACKGROUND_COLOR, json.common[themeName]);
   });
 });
+
+When('I open {string} component with theme {string} all buttons story', (componentName, themeName) => {
```

Comment:
Can we make more useable step which could be used also for other components with theme, not only for all buttons ?

Reply:
We will refactor this step, when we decide the strategy of naming of stories for `visual` tests